pyproject.toml

```diff
@@ -24,7 +24,7 @@ dependencies = [
   "pynacl==1.5.0",
   "cryptography==36.0.2",
   "mnemonic==0.20",
-  "multiversx-sdk-core==0.3.*"
```

Comment:
The only production code that this package depends on is `multiversx_sdk_core.Address`.